pkg/trace/obfuscate/obfuscate.go

```diff
@@ -42,21 +46,45 @@ func (o *Obfuscator) SQLLiteralEscapes() bool {
 	return atomic.LoadInt32(&o.sqlLiteralEscapes) == 1
 }
 
-// NewObfuscator creates a new obfuscator from the provided config
-func NewObfuscator(cfg *config.ObfuscationConfig) *Obfuscator {
+// NewObfuscator creates a new obfuscator
+func NewObfuscator(cfg *traceconfig.ObfuscationConfig) *Obfuscator {
 	if cfg == nil {
-		cfg = new(config.ObfuscationConfig)
+		cfg = new(traceconfig.ObfuscationConfig)
 	}
 	o := Obfuscator{opts: cfg}
 	if cfg.ES.Enabled {
-		o.es = newJSONObfuscator(&cfg.ES)
+		o.es = o.newJSONObfuscator(&cfg.ES)
 	}
 	if cfg.Mongo.Enabled {
-		o.mongo = newJSONObfuscator(&cfg.Mongo)
+		o.mongo = o.newJSONObfuscator(&cfg.Mongo)
+	}
+	if cfg.SQLExecPlan.Enabled {
+		o.sqlExecPlan = o.newJSONObfuscator(&cfg.SQLExecPlan)
+	}
+	if cfg.SQLExecPlanNormalize.Enabled {
+		o.sqlExecPlanNormalize = o.newJSONObfuscator(&cfg.SQLExecPlanNormalize)
 	}
 	return &o
 }
 
+// NewSQLObfuscator initializes the obfsucator from the standard agent config, setting defaults for SQLExecPlan
```

Comment:
Since this is only used in `pkg/collector/python`, should we move this function over there along with the defaults?

Reply:
I don't think so, because most of the stuff in `pkg/collector/python` is cgo wrapper code calling out to core logic is that elsewhere. So I think it fits pretty well in here.

Reply:
I don't like this because of two reasons:
1. It confuses the API by having the name `NewSQLObfuscator`, which can lead one to believe it's an SQL obfuscator, which it's not: it's in fact a span obfuscator which supports everything else (Redis, ES, etc) but has special behaviour for JSON obfuscators.
2. It's not used anywhere inside `pkg/trace`
3. Seems to belong in `pkg/collector/python`

Reply:
I still hold the above belief and am not sure I understand why this is here...

Reply:
On second thought, I agree. Moved it https://github.com/DataDog/datadog-agent/pull/5692/commits/ba27f472e93cdcbc8dbe0f9cd9eff28119dff4dd